.github/actions/oss-rules-of-play-action/entrypoint.sh

```diff
@@ -15,6 +15,13 @@ if [ "$FOSSTARS_VERSION" = "" ]; then
     exit 1
 fi
 
+if [ "$TOKEN" = "" ]; then
+    echo "Achtung! No token provided or it is empty!"
+    TOKEN_OPTION=""
```

Comment:
Do we exit if there is no token?

Reply:
No, the action will continue to run but without token.